internal/run/convert.go

```diff
@@ -22,6 +27,71 @@ func v3ReleaseFound(release string, cfg *action.Configuration) bool {
 	return false
 }
 
+// clientsetFromFile returns a ready-to-use client from a kubeconfig file
+func clientsetFromFile(path string) (*kubernetes.Clientset, error) {
+	config, err := clientcmd.LoadFromFile(path)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to load admin kubeconfig")
+	}
+
+	overrides := clientcmd.ConfigOverrides{Timeout: "15s"}
+	clientConfig, err := clientcmd.NewDefaultClientConfig(*config, &overrides).ClientConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create API client configuration from kubeconfig")
+	}
+
+	client, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create API client")
+	}
+	return client, nil
+}
+
+// getReleaseConfigmaps returns a list of configmaps that are helm v2 releases
+func getReleaseConfigmaps(clientset kubernetes.Interface, release, tillerNamespace, tillerLabel string) (*corev1.ConfigMapList, error) {
+
+	if tillerNamespace == "" {
+		tillerNamespace = "kube-system"
+	}
+	if tillerLabel == "" {
+		tillerLabel = "OWNER=TILLER"
+	}
+	if release != "" {
+		tillerLabel += fmt.Sprintf(",NAME=%s", release)
+	}
+
+	configMaps, err := clientset.CoreV1().ConfigMaps(tillerNamespace).List(metav1.ListOptions{
+		LabelSelector: tillerLabel,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return configMaps, nil
```

Comment:
**nit:** This could simply be:

```
return clientset.CoreV1().ConfigMaps(tillerNamespace).List(metav1.ListOptions{
	LabelSelector: tillerLabel,
})
```